yt_dlp/extractor/mzaalo.py

```diff
@@ -0,0 +1,96 @@
+from .common import InfoExtractor
+from ..utils import (
+    parse_age_limit,
+    parse_duration,
+    traverse_obj,
+    url_or_none,
+)
+
+
+class MzaaloIE(InfoExtractor):
+    _VALID_URL = r'https?://www.mzaalo.com/play/(?P<type>movie|original|clip)/(?P<id>[a-fA-F0-9-]+)/[\w-]+'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?mzaalo\.com/play/(?P<type>movie|original|clip)/(?P<id>[a-fA-F0-9-]+)/[\w-]+'
```
we do want to escape dots in hostnames, and make the subdomain optional